lbjava-nlp-tools/README.md

```diff
@@ -2,3 +2,12 @@
 
 A collection of NLP tools written as part of the original [LBJava](https://github.com/IllinoisCogComp/lbjava)
 project. Meant to be used in conjunction with LBJava. 
+
+## Citation
+
+Thank you for citing us if you use us in your work!
+
+```
+@inproceedings{
```

Comment:
This can cite: http://cogcomp.cs.illinois.edu/page/publication_view/208 (you can grab the bibitem from the url)